### ISSUE TITLE
Invalid Argument for HEALTHCHECK

### ISSUE THREAD

**Issue by jokay** [replicatedhq/dockerfilelint#166]
When using this HEALTHCHECK in a Dockerfile:

```sh
HEALTHCHECK --interval=30s --timeout=10s --start-period=15s --retries=5 \
  CMD wget --spider http://localhost/health || exit 1
```

I get this linting error from [super-linter](https://github.com/github/super-linter) / dockerfilelint **1.8.0**:

```sh
File:   /builds/dev/app.test/src/app.test.server/Dockerfile
Issues: 1
Line 60: HEALTHCHECK --interval=30s --timeout=10s --start-period=15s --retries=5 \
Issue  Category      Title                 Description
    1  Possible Bug  Invalid Argument      The arguments to this command are invalid
                     Format]
```

Any idea on how to fix this?

**Comment by jokay** [comment]
Related issue [#28](https://github.com/replicatedhq/dockerfilelint/issues/28) & pull request [#117](https://github.com/replicatedhq/dockerfilelint/pull/117).

**Comment by jokay** [comment]
@egoexpress any idea? :wink:

**Comment by jokay** [comment]
When removing the option `--start-period=15s` from the Dockerfile, there won't be any linting error anymore.

**Comment by egoexpress** [comment]
> 
> 
> When removing the option `--start-period=15s` from the Dockerfile, there won't be any linting error anymore.

Mmmh, ok, seems to be a bug. I'll have a look into it. '--start-period' is a valid option for HEALTHCHECK and is already included in the linting function, so it *should* work.

**Comment by egoexpress** [comment]
Found the issue. The regular expression checking the option names doesn't count for the dash in the '--start-period' option. PR #167 fixes this.

**Comment by jokay** [comment]
Thx for the fix.

**Comment by lvjp** [comment]
I missed up with issue number... It was #156....
